src/app/components/LiveLabel/__snapshots__/index.test.tsx.snap

```diff
@@ -232,7 +307,13 @@ exports[`LiveLabel should render correctly with localised live text 1`] = `
       class="emotion-1"
       dir="ltr"
     >
-      AS E DE HAPPEN 
+      As E Dey Happen 
+    </span>
+    <span
+      class="emotion-2"
+      lang="en-GB"
```

Comment:
Should not announce 'comma' with an english acsent please

Reply:
Only the word 'live'

Reply:
I guess this means we need to output the comma in a seperate span?

Reply:
Does it announce `comma`? I thought it causes the screenreader to pause? Let's see how tricky (or not) it is to fix... 🤔

Reply:
No it doesn't announce it when reading normally, you are correct it creates a pause, though on a screen reader you can read letter by letter and in that instance depending on your settings I would of thought that it would be announced... Not a biggie.